<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task</class>
 <widget class="QWidget" name="Task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>275</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pathsTab">
      <attribute name="title">
       <string>Paths</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeView" name="pathsTreeView">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layersTab">
      <attribute name="title">
       <string>Layers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeView" name="layersTreeView">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="moveUp">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resource/resource.qrc">
          <normaloff>:/icons/layer-raise.svg</normaloff>:/icons/layer-raise.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveDown">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resource/resource.qrc">
          <normaloff>:/icons/layer-lower.svg</normaloff>:/icons/layer-lower.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveTop">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resource/resource.qrc">
          <normaloff>:/icons/layer-top.svg</normaloff>:/icons/layer-top.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveBottom">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resource/resource.qrc">
          <normaloff>:/icons/layer-bottom.svg</normaloff>:/icons/layer-bottom.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
